### PR TITLE
feat: helm cluster scanner deployment

### PR DESCRIPTION
**Pull request by shmu3l**

### CODE REVIEW

File: charts/datree-admission-webhook/templates/cluster-scanner.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                  key: {{ .Values.datree.existingSecret.key }}
+            {{- else }}
+              value: "{{ .Values.datree.token }}"
+            {{- end }}
+            - name: DATREE_POLICY
+              value: {{.Values.datree.policy | default "Starter"}}
+            - name: DATREE_NAMESPACE
+              value: {{template "datree.namespace" .}}
+            - name: SCAN_INTERVAL_HOURS
+              value: "{{ now | dateModify "-5m" | date "04" }} */{{ .Values.datree.scanIntervalHours | default 1 }} * * *" # every hour, starting at least 55 minutes after helm installation
```
Comment (myishay) [inline charts/datree-admission-webhook/templates/cluster-scanner.yaml:104]: I think this should be just the number as-is.
The cron template was just a workaround.

File: charts/datree-admission-webhook/templates/cluster-scanner.yaml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+              path: /ready
+              port: 8444
+              scheme: HTTPS
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          image: "{{ .Values.clusterScanner.image.repository }}:{{ .Values.clusterScanner.image.tag | default "0.0.2" }}"
+          imagePullPolicy: "{{.Values.clusterScanner.image.pullPolicy}}"
+          resources: {{- toYaml .Values.clusterScanner.resources | nindent 12 }}
+          ports:
+            - containerPort: 8444
```
[the remark this answers is not in the document]
Comment (shmu3l) [reply charts/datree-admission-webhook/templates/cluster-scanner.yaml:127]: took it from the cluster-scanner server port 
[https://github.com/datreeio/cluster-scanner/blob/bbc0b633aab509d31ff39899b9a9bfa2b41dff66/main.go#L94](https://github.com/datreeio/cluster-scanner/blob/bbc0b633aab509d31ff39899b9a9bfa2b41dff66/main.go#L94)

[the remark this answers is not in the document]
Comment (myishay) [reply charts/datree-admission-webhook/templates/cluster-scanner.yaml:127]: got it, thanks

**Review by myishay — APPROVED** [review]

File: charts/datree-admission-webhook/templates/cluster-scanner.yaml [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                  key: {{ .Values.datree.existingSecret.key }}
+            {{- else }}
+              value: "{{ .Values.datree.token }}"
+            {{- end }}
+            - name: DATREE_POLICY
+              value: {{.Values.datree.policy | default "Starter"}}
+            - name: DATREE_NAMESPACE
+              value: {{template "datree.namespace" .}}
+            - name: SCAN_INTERVAL_HOURS
+              value: "0 */{{ .Values.datree.scanIntervalHours | default 1 }} * * *"
```
Comment (myishay) [inline charts/datree-admission-webhook/templates/cluster-scanner.yaml:110]: @shmu3l ask @royhadad

Comment (royhadad) [reply charts/datree-admission-webhook/templates/cluster-scanner.yaml:110]: I think @shalev007 used the variable "as-is"

**Pull request merged**